tests/utils/check-test-format

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python3
+
+import argparse
+import glob
+import os
+import sys
+import yaml
+import yamlordereddictloader
+
+SUPPORTED_KEYS = [
+    "summary",
+    "details",
+    "backends",
+    "systems",
+    "manual",
+    "priority",
+    "warn-timeout",
+    "kill-timeout",
+    "environment",
+    "prepare",
+    "restore",
+    "debug",
+    "execute",
+]
+MANDATORY_KEYS = ["summary", "execute"]
+
+
+def check_mandatory_keys(task_keys):
+    findings = []
+    for key in MANDATORY_KEYS:
+        if key not in task_keys:
+            findings.append("Key '{}' is mandatory".format(key))
+
+    return findings
+
+
+def check_keys_order(task_keys):
+    last_index = -1
+    last_key = ""
+    findings = []
+
+    for curr_key in task_keys:
+        try:
+            curr_index = SUPPORTED_KEYS.index(curr_key)
+            if curr_index <= last_index:
+                findings.append(
+                    "Keys '{}' and '{}' do not follow the desired order: {}".format(
+                        last_key, curr_key, SUPPORTED_KEYS
+                    )
+                )
+
+            last_index = curr_index
+            last_key = curr_key
+
+        except ValueError as err:
+            findings.append(
+                "key '{}' not included in the suported keys: {}".format(
```

Comment:
s/suported/supported/

Reply:
Though maybe "key X is not among the supported keys: ..." would be clearer.

Reply:
done, thanks for the suggestion